### PR TITLE
UI design changes

### PR DESCRIPTION
**Pull request by dbobe**
Changes in header and footer to be closer to prototype. Change in theme for colors and font. Changes in muiCard just colors.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,18 +1,13 @@
-import {
-  Link,
-  Paper,
-  Typography
-} from "@mui/material";
+import { Paper, Typography } from "@mui/material";
+import theme from "../theme";
```
Comment (NullDefault) [inline src/components/Footer.js]: is importing theme the correct way to do things? This seems like a hack
https://mui.com/customization/theming/

File: src/components/muiCard.js [hunk 2]
```diff
@@ -8,26 +8,21 @@ import {
...
+  mapLink,
 }) {
-
   const [starred, setStarred] = useState(false);
 
   return (
     <Card
       sx={{
-        bgcolor: "primary.main",
+        bgcolor: "secondary.main",
```
Comment (NullDefault) [inline src/components/muiCard.js:25]: cause you don't need to import theme here. I think you're just importing the JSON object and using its property. Maybe make a "style constants" file of some kind and keep the gradients there?

**Review by NullDefault — CHANGES_REQUESTED** [review]
Importing the theme object and then accessing its field is not the correct way to do things (I'm pretty sure)

File: src/components/Footer.js (hunk 1 above)
Comment (dbobe) [reply src/components/Footer.js]: Gradient was removed from header and footer

**Review by NullDefault — APPROVED** [review]
LGTM

**Pull request merged**